configs/bedrock.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=Expensify Bedrock Server
+Documentation=http://bedrockdb.com/
+After=network.target
+
+# The following creates the database file if absent
+After=bedrock_prerequisites.service
+Requires=bedrock_prerequisites.service
+
+[Service]
+EnvironmentFile=/etc/bedrock/bedrock.conf
+PIDFile=/var/run/bedrock.pid
+
+Type=forking
+
+ExecStart=/usr/sbin/bedrock \
+        -fork \
+        -pidFile /var/run/bedrock.pid \
+        -nodeName ${THISNODE} \
+        -db ${BEDROCK_DB_PATH} \
+        -serverHost ${SERVER_HOST} \
+        -nodeHost ${NODE_HOST} \
+        -priority ${PRIORITY} \
+        ${PLUGINS} \
+        ${WORKER_THREADS} \
```

Comment:
we should remove this

Reply:
This should stay, so the users can simply uncomment the worker threads option in the bedrock.conf file without having to also modify the service file.

Reply:
Ah that makes sense.